includes/Data/Brands.php

```diff
@@ -34,7 +34,7 @@ public static function get_config( Container $container ) {
 						'woocommerce_default_country' => 'AU:NSW',
 						'woocommerce_currency'        => 'AUD',
 					),
-					'wondercartBuyNow' => ''
+					'wondercartBuyNow' => '',
```

Comment:
We were to remove all `Brands.php` changes right?

Reply:
yeah, the changes were removed already